pymemcache/client/base.py

```diff
@@ -315,7 +317,8 @@ def set(self, key, value, expire=0, noreply=None):
         """
         if noreply is None:
             noreply = self.default_noreply
-        return self._store_cmd(b'set', {key: value}, expire, noreply)[key]
+        return self._store_cmd(b'set', {key: value}, expire, noreply,
+                               flags=flags)[key]
 
     def set_many(self, values, expire=0, noreply=None):
```

Comment:
Does it make sense for `set_many` to also accept `flags`?

Reply:
so `_store_cmd()` is being called by `add()`/ `append()`/ `cas()`/` prepend()`/` replace()`/ `set()`/ `set_many()`. I mean I can add flags to all those functions, but if there are no use cases consuming flags from these functions, adding flags might make it more confusing for the consumers.

Reply:
I think it would make sense to add them there in the long-term, but I'm comfortable waiting until people need them given that this is still a new capability.